collectors/ebpf.plugin/README.md

```diff
@@ -8,28 +8,24 @@ sidebar_label: "eBPF"
 
 # eBPF monitoring with Netdata
 
-eBPF consists of a wide toolchain that ultimately outputs a set of bytecode that will run inside the eBPF virtual
-machine (VM) which lives inside the Linux kernel. The program in particular is executed in response to a [tracepoint
-or kprobe](#probes-and-tracepoints) activation.
+The Netdata Agent provides many [eBPF](https://ebpf.io/what-is-ebpf/) programs to help you troubleshoot and debug how applications that interact with the Linux kernel. The `ebpf.plugin` uses [tracepoints or kprobes](#data-collection) to collect a wide array of high value data about the host that would otherwise be impossible to capture. 
 
-Netdata has written many eBPF programs, which, when compiled and integrated into the Netdata Agent, are able to collect
-a wide array of data about the host that would otherwise be impossible. The data eBPF programs can collect is truly unique,
-which gives the Netdata Agent access to data that is high value but normally hard to capture.
+>❗ eBPF monitoring only works on Linux systems and with specific Linux kernels, including all kernels newer than `4.11.0`, and all kernels on CentOS 7.6 or later. For kernels older than `4.11.0`, improved support is in active development.
 
-eBPF monitoring can help you troubleshoot and debug how applications interact with the Linux kernel. See
-our [guide on troubleshooting apps with eBPF metrics](/docs/guides/troubleshoot/monitor-debug-applications-ebpf.md) for
-configuration and troubleshooting tips.
+This document provides all details about the `ebpf.plugin`.
+For hands-on configuration and troubleshooting tips see our [tutorial on troubleshooting apps with eBPF metrics](/docs/guides/troubleshoot/monitor-debug-applications-ebpf.md).
 
 <figure>
   <img src="https://user-images.githubusercontent.com/1153921/74746434-ad6a1e00-5222-11ea-858a-a7882617ae02.png" alt="An example of VFS charts, made possible by the eBPF collector plugin" />
-  <figcaption>An example of VFS charts made possible by the eBPF collector plugin.</figcaption>
+  <figcaption>An example of virtual file system (VFS) charts made possible by the eBPF collector plugin.</figcaption>
 </figure>
 
-## Probes and Tracepoints
+<a id="data-collection"> </a>
+## How Netdata collects data using probes and tracepoints
 
-The following two features from the Linux kernel are used by Netdata to run eBPF programs:
+Netdata uses the following two features from the Linux kernel to run eBPF programs:
 
-- Kprobes and return probes (kretprobe): Probes can insert virtually into any kernel instruction. When eBPF runs in
+- Kprobes and return probes (`kretprobe`): Probes can insert virtually into any kernel instruction. When eBPF runs in
```

Comment:
We also use [trampoline](https://lwn.net/Articles/804937/), but we are still integrating it with our documentation, should we wait the integration to finish? Or is it better to wait  we merge everything?

Reply:
@thiagoftsm 
1. Could you clarify what you mean by "you're still integrating it with the documentation"? 
2. If you feel ready, please add a short explanation about trampoline in line 30. :)

Reply:
1. We did not add trampoline for all threads, so this is still a WIP, and in normal situation we update documentation when the feature is merged.
2. I pushed a commit adding trampolines, because we already have them for some threads.

Reply:
Thanks, that's good to know!